intermediate-exercises/src/components/02-StopWatch.js

```diff
@@ -1,26 +1,51 @@
 import React, { Component } from 'react';
 
-/*
-* Exercise 2:
-*
-*  Create a `StopWatch` component that has a Start/Stop button and a Clear
-*  button.
-*
-*  Pressing Start will start a timer and the lapsed time in
-*  milliseconds should be displayed above the buttons.
-*
-*  Once started the Start button should change to Stop. Clicking Stop
-*  will stop the timer but lapsed time will be preserved.
-*
-*  Clicking Start again will resume the timer from where it left off.
-*
-*  Clicking Clear will stop the timer if it's running and reset the lapsed time to 0.
-*/
 
 class StopWatch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      time: 0,
+      isTimerOn: false,
+    }
+    this.startTimer = this.startTimer.bind(this);
+    this.stopTimer = this.stopTimer.bind(this);
+    this.resetTimer = this.resetTimer.bind(this);
+  }
+  startTimer() {
+    return this.setState({
+      isTimerOn: true,
+    }),
+    this.timer = setInterval(() => {
+        this.setState({
+          time: this.state.time + 1,
```

Comment:
The time should increase by milliseconds as instructed in the task, and not with seconds. Also refreshing it after every one second will show a slow timer in milliseconds. Timeout value should be smaller.